adapter/sink/README.md

```diff
@@ -0,0 +1,3 @@
+# notification-adapter
+
+Notification-adapter receive notifications from notifcation manager and export with the socket.
```

Comment:
=> and then send it to a TCP endpoint.

Reply:
Maybe we can add more instructions here on how to use it together with the Notification Manager?